engine/lite/kafka/runtime/src/main/scala/pl/touk/nussknacker/engine/lite/kafka/HealthCheckServerRunner.scala

```diff
@@ -0,0 +1,59 @@
+package pl.touk.nussknacker.engine.lite.kafka
+
+import akka.actor.{Actor, ActorSystem, Props}
+import akka.http.scaladsl.model.Uri
+import akka.management.scaladsl.AkkaManagement
+import akka.pattern._
+import akka.util.Timeout
+import com.typesafe.scalalogging.LazyLogging
+import pl.touk.nussknacker.engine.lite.kafka.KafkaScenarioInterpreterStatusCheckerActor.GetStatus
+import pl.touk.nussknacker.engine.lite.kafka.TaskStatus.{Running, TaskStatus}
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+class HealthCheckServerRunner(system: ActorSystem, scenarioInterpreter: KafkaTransactionalScenarioInterpreter) {
```

Comment:
Cannot help thinking it's a bit of an overkill, but let's leave it :)

Reply:
Why overkill? It is very light library - it depends only on akka-http. Amount of code is less or equal to writting it on our side. Benefits are: 
- we can easily configure combination of checks in application.conf
- we stick to some akka standard solution instead of writting our new one (standard port, urls, we can use some standard checks), good documentation: https://doc.akka.io/docs/akka-management/current/index.html#overview
- we can turn on security if we want
- some additional extensions available that could be useful
- it will be easy to switch to our own made solution

Reply:
ok ;) :+1: